src/DetailsView/reports/components/automated-check-issue-details-card.scss

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+@import '../../../common/styles/colors.scss';
+@import '../../../common/styles/fonts.scss';
+
+.base-card-layer-main {
+    position: relative;
+    opacity: 1;
+    background: #fff;
+    padding: 20px;
+    border-radius: 4px;
+    box-shadow: 0px 0.6px 1.8px rgba(0, 0, 0, 0.108), 0px 3.2px 7.2px rgba(0, 0, 0, 0.132);
+}
+
+.report-instance-table {
+    font-family: Menlo, Consolas, Courier New, monospace;
+    word-break: break-all;
+}
+
+.report-instances .row {
+    top: calc(50% - 20px / 2);
+    border-bottom: 0.5px solid $neutral-10;
+}
+
+.report-instances .row:last-child {
+    border-bottom: none;
+}
+
+.report-instance-table .label {
+    font-family: Segoe UI;
+    font-size: 14px;
+    line-height: 20px;
+    font-weight: bold;
+    color: $primary-text;
+    text-align: left;
+    width: 120px;
+}
+.report-instance-table .content {
+    color: $secondary-text;
+    margin: 12px 12px 12px 0px;
+    font-size: 14px;
+    line-height: 20px;
+    align-items: flex-end;
+    display: flex;
+}
+
+table.report-instance-table {
+    border-collapse: collapse;
+    overflow-x: auto;
+}
+
+@media screen and (max-width: 600px) {
```

Comment:
does `600px` here means that's the breaking point? i.e if the window size is less than this we'll see the "compact" version of the card?

Reply:
Yes. less than 600px the compact version shows up

Reply:
Base on the figma file, it should be `480px`